app/helpers.js

```diff
@@ -1,10 +1,12 @@
+const { DEFAULT_OFFSET, DEFAULT_LIMIT, DEFAULT_SORTKEY, TEXT_DESC, ASC, DESC } = require('./constants');
+
 exports.paginated = (data, params) => {
-  const { offset = 0, limit = 5 } = params;
+  const { offset = DEFAULT_OFFSET, limit = DEFAULT_LIMIT } = params;
```

Comment:
Utils should be as dummy as possible. Please, use the default params in the service

Reply:
changes requested ok, additionally changes the default to albums/queries.js